AnalyticsRallyHere/Source/AnalyticsRallyHere/Private/RH_AnalyticsProvider.cpp

```diff
@@ -28,7 +28,7 @@ TSharedPtr<IAnalyticsProvider> FRH_Analytics::CreateAnalyticsProvider(const FAna
 	// If we didn't have a proper APIServer, return NULL
 	if (ConfigValues.APIServerET.IsEmpty())
 	{
-		UE_LOG(LogAnalytics, Warning, TEXT("CreateAnalyticsProvider config not contain required parameter %s"), *FAnalyticsET::Config::GetKeyNameForAPIServer());
+		UE_LOG(LogAnalytics, Warning, TEXT("[RHAnalytics] CreateAnalyticsProvider config not contain required parameter %s"), *FAnalyticsET::Config::GetKeyNameForAPIServer());
```

Comment:
Shouldn't we make our own log source, instead of prefixing all of the logs?

Reply:
Yeah - the ET one is inconsistent about it, but uses a prefix similar to the above, but a dedicated log channel would be better anyways.